config/tests/guest/libvirt/upstream.cfg

```diff
@@ -0,0 +1,251 @@
+                threads_vm3=4
+                sockets_vm3=1
+                vcpu_cores_vm3=2
+                vcpu_threads_vm3=4
+                vcpu_sockets_vm3=1
+                cores_vm4=1
+                threads_vm4=8
+                sockets=1
+                vcpu_cores_vm4=1
+                vcpu_threads_vm4=8
+                vcpu_sockets_vm4=1
+                virtinstall_qemu_cmdline_vm1=" -M pseries,vsmt=1"
+                virtinstall_qemu_cmdline_vm2=" -M pseries,vsmt=2"
+                virtinstall_qemu_cmdline_vm3=" -M pseries,vsmt=4"
+                virtinstall_qemu_cmdline_vm4=" -M pseries,vsmt=8"
+            - mem_merge:
+                no with_intc,with_vsmt
+                only with_smt2
+                variants:
+                    - enable:
+                        virtinstall_qemu_cmdline_vm1=" -M pseries,ic-mode=dual,mem-merge=True"
+                        virtinstall_qemu_cmdline_vm2=" -M pseries,ic-mode=xive,mem-merge=True"
+                        virtinstall_qemu_cmdline_vm3=" -M pseries,ic-mode=xics,mem-merge=True"
+                        virtinstall_qemu_cmdline_vm4=" -M pseries,ic-mode=dual,max-cpu-compat=power8,mem-merge=True"
+                        virtinstall_qemu_cmdline_vm5=" -M pseries,ic-mode=dual,kernel-irqchip=off,mem-merge=True"
+                        virtinstall_qemu_cmdline_vm6=" -M pseries,ic-mode=xive,kernel-irqchip=off,mem-merge=True"
+                        virtinstall_qemu_cmdline_vm7=" -M pseries,ic-mode=xics,kernel-irqchip=off,mem-merge=True"
+                        virtinstall_qemu_cmdline_vm8=" -M pseries,ic-mode=dual,max-cpu-compat=power8,kernel-irqchip=off,mem-merge=True"
+                   - disable:
+                        virtinstall_qemu_cmdline_vm1=" -M pseries,ic-mode=dual,mem-merge=False"
+                        virtinstall_qemu_cmdline_vm2=" -M pseries,ic-mode=xive,mem-merge=False"
+                        virtinstall_qemu_cmdline_vm3=" -M pseries,ic-mode=xics,mem-merge=False"
+                        virtinstall_qemu_cmdline_vm4=" -M pseries,ic-mode=dual,max-cpu-compat=power8,mem-merge=False"
+                        virtinstall_qemu_cmdline_vm5=" -M pseries,ic-mode=dual,kernel-irqchip=off,mem-merge=False"
+                        virtinstall_qemu_cmdline_vm6=" -M pseries,ic-mode=xive,kernel-irqchip=off,mem-merge=False"
+                        virtinstall_qemu_cmdline_vm7=" -M pseries,ic-mode=xics,kernel-irqchip=off,mem-merge=False"
+                        virtinstall_qemu_cmdline_vm8=" -M pseries,ic-mode=dual,max-cpu-compat=power8,kernel-irqchip=off,mem-merge=False"
+            #- with_nested_cap:
+            #    no with_vsmt,with_intc,mem_merge
+            #    only with_smt4
+            #    virtinstall_qemu_cmdline_vm1=" -M pseries,cap-nested-hv=True"
+        variants:
+            - with_virtio_scsi:
+                kernel_args='root=/dev/sda2 rw console=tty0 console=ttyS0,115200 init=/sbin/init initcall_debug selinux=0'
```

Comment:
are we sure it always populate with /sda2 i am pointing this if it not there may test will fail can we handle this dynamic way

Reply:
yep, this will work with JeOS...